### DISCUSSION TITLE
Size difference between two machines when filer.sync is ran

### DISCUSSION THREAD

**Discussion by jonnythebard** [seaweedfs/seaweedfs#6162] [Q&A]
Hi. I am using filer.sync and there is a volume size difference.

I have 2 machines that runs seaweedfs. One is for the primary and another is for secondary. The primary is running seaweedfs with commit hash `06434f5227d6ee89d5f7bc3367132cec9b9fdb61` and the secondary `82ed61c6552e5095c682131012c91594ad2643e6`. 
I have 28 volumes running at each machine and using redis as filer database. Also, the primary has been storing 8T * 28 size of data for past 10 months while the secondary one was added to my cluster like a week ago.

From the secondary machine, the following command is running.
`weed filer.sync -a primary:8888 -b localhost:8888 -isActivePassive --concurrency=1024`

My filer.sync has almost caught up with the primary's recent changes and I expected both machine to have about same size.
But turns out the secondary only has like 5.7T * 28 while the primary is 8T * 28.
as I ran `DBSIZE` at redis-clie at each of machine, I get 8784694 for the primary machine and 8525182 for the secondary.

I haven't found any file broken because it's too big to check one by one.. Are these differences kinda expected for some reason?

**Comment by jonnythebard** [comment]
I think some files are not synced because the downloading crashed.
I attached a log file.
Is there any way to find crashed files and run the sync again?
[E1024 00:58:44.280870 filer_sync_jobs.log](https://github.com/user-attachments/files/17505731/E1024.00.58.44.280870.filer_sync_jobs.log)

**Reply by chrislusf** [reply]
This could be a bug related to manifest chunk handling during filer.sync.

Can the original file be read out successfully? Copying the file directly could be a temp fix.

**Reply by chrislusf** [reply]
based on the git version, I suspect you may hit https://github.com/seaweedfs/seaweedfs/issues/5276
please consider upgrading first.
